src/Form.svelte

```diff
@@ -104,8 +104,10 @@
               <span>
                 <a class="link" href="{field.value}" target="_blank">{field.value}</a>
               </span>
+            {:else if field.name === "owner"}
+              <Address address={field.value} resolve {config} />
             {:else if isAddress(field.value)}
-              <Address address={field.value} {config} />
+              <Address noAvatar address={field.value} {config} />
```

Comment:
Any reason we're not showing avatars anymore here?

Reply:
I removed the avatars, since the place we wanted to show avatars was in the owner field.
We can eventually add a `noAvatar` attribute to fields were we don't want to show avatar eg: in the address field in the `Registrations::View`.